osquery/tables/system/windows/registry.cpp

```diff
@@ -72,6 +72,21 @@ void explodeRegistryPath(const std::string& path,
   rKey = osquery::join(toks, kRegSep);
 }
 
+Status sanitizeRegistryStrings(const DWORD dataType,
+                               BYTE* dataBuff,
+                               DWORD dataSize) {
+  auto status = Status(0, "OK");
```

Comment:
Rather than creating a `status` variable, what about just returning `Status(0, "Ok");` and `Status(1, "Fail");`?